lib/3scale/backend/worker_async.rb

```diff
@@ -1,4 +1,6 @@
 require 'async'
+require 'async/barrier'
```

Comment:
I guess now we don't need to require 'async/barrier'

Reply:
Right!. Done.